src/partials/about.html

```diff
@@ -17,8 +17,8 @@ <h1>How it's made?</h1>
     </div>
     <button type="button" class="read-more-btn">
       Read more
-      <svg class="icon sec-features__icon" width="5px" height="8px">
-        <use href="./src/images/icons.svg#icon-icon-greater"></use>
+      <svg class="icon" width="5px" height="8px">
+        <use href="./images/icons.svg#icon-icon-greater"></use>
```

Comment:
Please pay attention to the href attribute. Use example.html to find out the correct example, like this -  `"images/logo.png"`.